### ISSUE TITLE
Translate the Store information (Mozilla, Chrome, Edge, Ubuntu) into at least French and Spanish

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js#1118]
While I believe a lot of Stores provide automatic translation, at least the output should be checked for the languages of the app (i.e. French and Spanish, as well as English, for now). If the translations are poor, we should take control of them and produce our own translations.

**Comment by Jaifroid** [comment]
* Edge Store shows localized UI in Spanish, but not the description text, which is in English.

**Comment by Jaifroid** [comment]
In sum, there is no automatic translation provided by the stores.

**Comment by Jaifroid** [comment]
Translations completed:

- [ ] Firefox Spanish - https://addons.mozilla.org/es/firefox/addon/kiwix-offline/
- [ ] Firefox French
- [ ] Edge Spanish
- [ ] Edge French
- [ ] Chrome Spanish
- [ ] Chrome French

**Comment by Jaifroid** [comment]
To translate Chrome and Edge Store entries, it is necessary to add localization strings to the app's manifest or in a _localization folder. So this has to be completed before the stores can be updated. The main thing to translate in _localizaiton is the app description, and one or two other general fields, as we don't use that extension-only localization solution for i18n in-app.

**Comment by Bam92** [comment]
@Jaifroid,
I'm available to provide french translation for Chrome. Can make a PR for just that?

**Comment by Jaifroid** [comment]
@Bam92 The problem is that I don't currently have direct access to the Chrome Store in order to be able to upload a French (or any other language) translation of the info. I already did the Mozilla one, where I do have direct access. Please note that making these translations is not about making a PR on this Repo, because the Store info is only available on the Store page, so if you're looking to get some PRs under your belt, this wouldn't be such a useful issue for you to tackle. But thanks for the offer!

**Comment by Jaifroid** [comment]
Note that there is no way (that I can find) to provide separate French or Spanish store info for Ubuntu Touch. So this item is complete if the Chrome and Edge stores accept the translated information (currently under review).
